styles.css

```diff
@@ -6,7 +10,7 @@
 
 body {
   width: 100%;
-  height: 100%;
+  height: 100vh;
 
   /* background: #E5E5E5; */
```

Comment:
- [ ] At exactly 768px, I observed that all the elements on the UI were not displayed on the screen as expected, though there is an horizontal overflow but this doesn't enhance good user experience. Kindly ensure that you app is perfectly responsive without horizontal overflow. This [tutorial](https://www.tutorialspoint.com/css/css_responsive.htm) can be helpful. ℹ️ 

<img width="514" alt="Untitled" src="https://github.com/tan12082001/Project-portfolio/assets/103386183/6d82ea37-d514-49b0-bca0-cfa06bc341b9">

- [ ] Kindly remove this commented line of code; this wil make your codebase clean. 👍

Reply:
> - [ ] Hello @tan12082001, As the previous code reviewer asked, could you remove the comment line of code on line 15? This will make your codebase clean. 👍